### PR TITLE
replace hex values with design tokens

### PR DESCRIPTION
**Pull request by garrettbear**
## **Description**

Replace static hex values to a token from design-tokens that best matches. If unavailable, a temporary file has been set up for any new tokens needed but not yet provided by the design-tokens package.

## **Related issues**

Fixes: #22363 

## **Manual testing steps**

1. Run locally
2. Compare [local storybook](http://localhost:6006/?path=/docs/getting-started-introduction--docs) to [production storybook](https://metamask.github.io/metamask-storybook/?path=/story/getting-started-introduction--docs) on the components color change
3. Be sure to test light and dark mode

## **Screenshots/Recordings**
 `See code comments`

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've clearly explained what problem this PR is solving and how it is solved.
- [x] I've linked related issues
- [ ] I've included manual testing steps
- [x] I've included screenshots/recordings if applicable
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [x] I’ve properly set the pull request status:
  - [x] In case it's not yet "ready for review", I've set it to "draft".
  - [x] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [x] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [x] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: ui/components/ui/icon/preloader/preloader-icon.component.js
[the remark this answers is not in the document]
Comment (georgewrmarshall) [reply ui/components/ui/icon/preloader/preloader-icon.component.js:1]: non-blocking: I suppose there is no `color` prop for this component so it's not wired up to the controls. Other option is to use `primary/default` and `primary/muted` here

File: ui/pages/settings/settings-tab/index.scss [hunk 1]
```diff
@@ -13,11 +13,11 @@
   }
 
   &__button--red {
-    border-color: lighten($monzo, 10%);
+    border-color: var(--color-error-muted);
     color: var(--color-error-default);
 
     &:active {
-      background: lighten($monzo, 55%);
+      background: var(--color-error-muted-pressed);
```
[the remark this answers is not in the document]
Comment (georgewrmarshall) [reply ui/pages/settings/settings-tab/index.scss:20]: Looks like that className is used here so good to update https://github.com/MetaMask/metamask-extension/blob/8782d2c767f01fb50644eff66da5abb21831ab26/ui/pages/settings/advanced-tab/advanced-tab.component.js#L222

File: ui/components/ui/icon/preloader/preloader-icon.component.js [hunk 2]
```diff
@@ -30,13 +30,13 @@ const Preloader = ({ className, size }) => (
         fillRule="evenodd"
         clipRule="evenodd"
         d="M8 13.7143C4.84409 13.7143 2.28571 11.1559 2.28571 8C2.28571 4.84409 4.84409 2.28571 8 2.28571C11.1559 2.28571 13.7143 4.84409 13.7143 8C13.7143 11.1559 11.1559 13.7143 8 13.7143ZM8 16C3.58172 16 0 12.4183 0 8C0 3.58172 3.58172 0 8 0C12.4183 0 16 3.58172 16 8C16 12.4183 12.4183 16 8 16Z"
-        fill="#037DD6"
+        fill="var(--brand-colors-blue-blue500)"
```
Comment (georgewrmarshall) [inline ui/components/ui/icon/preloader/preloader-icon.component.js:33]:
```suggestion
        fill="var(--color-primary-default)"
```

File: ui/components/ui/icon/preloader/preloader-icon.component.js [hunk 3]
```diff
@@ -30,13 +30,13 @@ const Preloader = ({ className, size }) => (
...
       />
     </mask>
     <g mask="url(#mask0)">
       <path
         d="M6.85718 17.9999V11.4285V8.28564H-4.85711V17.9999H6.85718Z"
-        fill="#037DD6"
+        fill="var(--brand-colors-blue-blue500)"
```
Comment (georgewrmarshall) [inline ui/components/ui/icon/preloader/preloader-icon.component.js:39]:
```suggestion
        fill="var(--color-primary-default)"
```

**Review by georgewrmarshall — COMMENTED** [review]
Looking good! One suggestion for the `Preloader` component
- Checked for no instances of deprecated colors ✅ 

Screenshot 2024-01-23 at 8 52 15 AM
Screenshot 2024-01-23 at 8 52 21 AM
Screenshot 2024-01-23 at 8 52 33 AM
Screenshot 2024-01-23 at 8 52 46 AM

**Review by georgewrmarshall — APPROVED** [review]
LGTM 🚀

**Review by darkwing — APPROVED** [review]

**Pull request merged**